sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -224,7 +224,8 @@
     <dependency>
       <groupId>net.minidev</groupId>
       <artifactId>json-smart</artifactId>
-      <version>2.3</version> <!-- {x-version-update;net.minidev:json-smart;external_dependency} -->
+      <!-- TODO: add {x-version-update;net.minidev:json-smart;external_dependency} this version tag below as part of PR: https://github.com/Azure/azure-sdk-for-java/pull/20770-->
```

Comment:
DIsabling this update.
As spring team aims to update this via this PR (#20770) post their releases.